packages/firebase_ui_localizations/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Should we add the default Licensing from Google here?

Reply:
done